os/drivers/mipidsi/mipi_dsi.h

```diff
@@ -0,0 +1,102 @@
+/****************************************************************************
+ *
+ * Copyright 2024 Samsung Electronics All Rights Reserved.
```

Comment:
Don't you get it from other opensource?
If you did, you should keep original license and should add Samsung license on top of it.
Could you check this?